Lib/collections/__init__.py

```diff
@@ -973,10 +973,17 @@ def __delitem__(self, key): del self.data[key]
     def __iter__(self):
         return iter(self.data)
 
-    # Modify __contains__ to work correctly when __missing__ is present
+    # Modify __contains__ and get() to work like dict
+    # does when __missing__ is present.
     def __contains__(self, key):
         return key in self.data
 
+    def get(self, key, default=None):
```

Comment:
Add a comment that *get()* is overridden to avoid a False positive when \__missing__ is defined.